packages/cli/src/commands/dev-contracts.ts

```diff
@@ -89,6 +89,7 @@ const commandModule: CommandModule<typeof devOptions, InferredOptionTypes<typeof
             saveDeployment: true,
             worldAddress,
             srcDir,
+            salt: undefined,
```

Comment:
I think it might be nice to do our own internally incrementing counter for this process here, so that restarting the dev process always results in the same contract address.

Reply:
Nevermind, this might be harder than I initially thought it would be. Since we already pass the `worldAddress` in to upgrade worlds, this should be fine.

Just a note that this means that our tests that generate data will be non-deterministic for now (unless we update them to pass in a consistent salt)